### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by mejlholm**

### CODE REVIEW

File: docs/src/main/asciidoc/cdi.adoc [hunk 1]
```diff
@@ -100,8 +100,8 @@ public class Translator {
 == _Q: Can I use setter and constructor injection?_
 
 A: Yes, you can.
-In fact, in CDI the "setter injection" is superseded by more powerful https://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#initializer_methods[initializer methods, window="_blank"].
-Intializers may accept multiple parameters and don't have to follow the JavaBean naming conventions.
+In fact, in CDI the "setter injection" is superseeded by more powerful https://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#initializer_methods[initializer methods, window="_blank"].
```
Comment (gsmet) [inline docs/src/main/asciidoc/cdi.adoc:103]: This change is incorrect.

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks for this. Just one issue: `superseded` is correct.

File: docs/src/main/asciidoc/cdi.adoc (hunk 1 above)
Comment (mejlholm) [reply docs/src/main/asciidoc/cdi.adoc]: You are right, I've reverted that change.

**Review by gsmet — APPROVED** [review]
Thanks!

**Review by geoand — APPROVED** [review]

**Pull request merged**